### ISSUE TITLE
Regression in conda-forge Windows CI due to protobuf migration

### ISSUE THREAD

**Issue by traversaro** [robotology/robotology-superbuild#639]

**Comment by traversaro** [comment]
Since five days ago, the nightly job on conda-forge on Windows is failing with:
~~~
2021-02-22T02:35:32.6734971Z     Checking Build System
2021-02-22T02:35:32.7422278Z     Building Custom Rule D:/a/robotology-superbuild/robotology-superbuild/src/GazeboYARPPlugins/plugins/multicamera/CMakeLists.txt
2021-02-22T02:35:32.8919598Z     MultiCameraPlugin.cpp
2021-02-22T02:35:33.9169693Z C:\Miniconda\envs\test\Library\include\ignition\msgs5\ignition/msgs/actor.pb.h(12,1): fatal error C1189: #error:  This file was generated by a newer version of protoc which is [D:\a\robotology-superbuild\robotology-superbuild\build\src\GazeboYARPPlugins\plugins\multicamera\gazebo_yarp_MultiCameraPlugin.vcxproj] [D:\a\robotology-superbuild\robotology-superbuild\build\GazeboYARPPlugins.vcxproj]
2021-02-22T02:35:34.0816311Z     Building Custom Rule D:/a/robotology-superbuild/robotology-superbuild/src/GazeboYARPPlugins/libraries/singleton/CMakeLists.txt
2021-02-22T02:35:34.1667061Z     Handler.cc
2021-02-22T02:35:35.4613055Z C:\Miniconda\envs\test\Library\include\ignition\msgs5\ignition/msgs/actor.pb.h(12,1): fatal error C1189: #error:  This file was generated by a newer version of protoc which is [D:\a\robotology-superbuild\robotology-superbuild\build\src\GazeboYARPPlugins\libraries\singleton\gazebo_yarp_singleton.vcxproj] [D:\a\robotology-superbuild\robotology-superbuild\build\GazeboYARPPlugins.vcxproj]
~~~

Last working: https://github.com/robotology/robotology-superbuild/actions/runs/585472203 (6 days ago)
First not working: https://github.com/robotology/robotology-superbuild/actions/runs/587714804 (5 days ago)

**Comment by traversaro** [comment]
Packages installed by last working:
~~~
2021-02-21T02:12:07.8441366Z   ace                          7.0.0  h0e60522_1           conda-forge/win-64       2 MB
2021-02-21T02:12:07.8441891Z   asio                        1.14.1  h0e60522_1           conda-forge/win-64     327 KB
2021-02-21T02:12:07.8443409Z   boost                       1.74.0  py39hefe7e4c_3       conda-forge/win-64     570 KB
2021-02-21T02:12:07.8444225Z   boost-cpp                   1.74.0  h54f0996_2           conda-forge/win-64      16 MB
2021-02-21T02:12:07.8444931Z   bzip2                        1.0.8  h8ffe710_4           conda-forge/win-64     Cached
2021-02-21T02:12:07.8445544Z   ca-certificates          2021.1.19  haa95532_0           pkgs/main/win-64       122 KB
2021-02-21T02:12:07.8446678Z   certifi                  2020.12.5  py39hcbf5309_1       conda-forge/win-64     144 KB
2021-02-21T02:12:07.8447375Z   console_bridge               1.0.1  h5362a0b_0           conda-forge/win-64      23 KB
2021-02-21T02:12:07.8448744Z   cppzmq                       4.7.1  h50e1b43_1           conda-forge/win-64      26 KB
2021-02-21T02:12:07.8449335Z   curl                        7.71.1  h4b64cdc_8           conda-forge/win-64     122 KB
2021-02-21T02:12:07.8449923Z   dlfcn-win32                  1.2.0  he025d50_0           conda-forge/win-64      21 KB
2021-02-21T02:12:07.8450528Z   eigen                        3.3.9  h2d74725_1           conda-forge/win-64     970 KB
2021-02-21T02:12:07.8451250Z   ffmpeg                       4.3.1  ha925a31_0           conda-forge/win-64      26 MB
2021-02-21T02:12:07.8452201Z   freeglut                     3.2.1  h0e60522_2           conda-forge/win-64     149 KB
2021-02-21T02:12:07.8452840Z   freeimage                   3.18.0  h33f27b4_0           conda-forge/win-64       2 MB
2021-02-21T02:12:07.8453676Z   freetype                    2.10.4  h546665d_1           conda-forge/win-64     489 KB
2021-02-21T02:12:07.8454413Z   gazebo                      11.3.0  h4f5d396_6           conda-forge/win-64      62 MB
2021-02-21T02:12:07.8455010Z   glew                         2.1.0  h39d44d4_2           conda-forge/win-64     941 KB
2021-02-21T02:12:07.8455528Z   glfw                         3.3.2  h2fa13f4_1           conda-forge/win-64     118 KB
2021-02-21T02:12:07.8456322Z   gsl                            2.6  hdfb1a43_2           conda-forge/win-64       2 MB
2021-02-21T02:12:07.8456929Z   gts                          0.7.6  h39c6fae_2           conda-forge/win-64     188 KB
2021-02-21T02:12:07.8457520Z   hdf5                        1.10.6  nompi_h5268f04_1114  conda-forge/win-64      20 MB
2021-02-21T02:12:07.8458093Z   icu                           68.1  h0e60522_0           conda-forge/win-64      16 MB
2021-02-21T02:12:07.8458791Z   ilmbase                      2.5.5  h12d4b20_0           conda-forge/win-64     294 KB
2021-02-21T02:12:07.8459442Z   intel-openmp                2020.3  h57928b3_311         conda-forge/win-64       2 MB
2021-02-21T02:12:07.8460011Z   ipopt                       3.13.3  hf6be2e5_8           conda-forge/win-64       1 MB
2021-02-21T02:12:07.8460794Z   irrlicht                     1.8.4  hb2b2432_0           conda-forge/win-64       1 MB
2021-02-21T02:12:07.8461431Z   jasper                      2.0.14  h77af90b_2           conda-forge/win-64     662 KB
2021-02-21T02:12:07.8462236Z   jpeg                            9d  he774522_0           conda-forge/win-64     344 KB
2021-02-21T02:12:07.8462920Z   jsoncpp                      1.9.4  h2d74725_1           conda-forge/win-64     963 KB
2021-02-21T02:12:07.8463495Z   krb5                        1.17.2  hbae68bd_0           conda-forge/win-64     Cached
2021-02-21T02:12:07.8464015Z   libblas                      3.9.0  8_mkl                conda-forge/win-64       4 MB
2021-02-21T02:12:07.8464606Z   libcblas                     3.9.0  8_mkl                conda-forge/win-64       4 MB
2021-02-21T02:12:07.8465333Z   libclang                    11.0.1  default_h5c34c98_1   conda-forge/win-64      21 MB
2021-02-21T02:12:07.8465974Z   libcurl                     7.71.1  h4b64cdc_8           conda-forge/win-64     Cached
2021-02-21T02:12:07.8466573Z   libignition-cmake2           2.6.2  h0e60522_1           conda-forge/win-64     175 KB
2021-02-21T02:12:07.8467234Z   libignition-common3         3.10.1  he3878d5_0           conda-forge/win-64     545 KB
2021-02-21T02:12:07.8468071Z   libignition-fuel-tools4      4.3.0  h0ee6c32_1           conda-forge/win-64     211 KB
2021-02-21T02:12:07.8469497Z   libignition-math6            6.7.0  h0e60522_1           conda-forge/win-64     198 KB
2021-02-21T02:12:07.8470411Z   libignition-msgs5            5.3.0  h0e60522_4           conda-forge/win-64       2 MB
2021-02-21T02:12:07.8471360Z   libignition-tools1           1.0.0  h57928b3_3           conda-forge/win-64       8 KB
2021-02-21T02:12:07.8472325Z   libignition-transport8       8.1.0  h7435cc3_3           conda-forge/win-64     326 KB
2021-02-21T02:12:07.8473956Z   libjpeg-turbo                2.0.5  he774522_0           conda-forge/win-64       1 MB
2021-02-21T02:12:07.8475208Z   liblapack                    3.9.0  8_mkl                conda-forge/win-64       4 MB
2021-02-21T02:12:07.8475756Z   liblapacke                   3.9.0  8_mkl                conda-forge/win-64       4 MB
2021-02-21T02:12:07.8476399Z   libmatio                    1.5.19  h216f7ef_0           conda-forge/win-64     156 KB
2021-02-21T02:12:07.8476927Z   libode                        0.16  h6538335_0           conda-forge/win-64     311 KB
2021-02-21T02:12:07.8477810Z   libopencv                    4.5.1  py39h27d8466_0       conda-forge/win-64      46 MB
2021-02-21T02:12:07.8478383Z   libpng                      1.6.37  ha81a0f5_2           conda-forge/win-64     763 KB
2021-02-21T02:12:07.8478980Z   libprotobuf                 3.14.0  h7755175_0           conda-forge/win-64       2 MB
2021-02-21T02:12:07.8479554Z   libsdformat                  9.3.0  h4bf3a07_2           conda-forge/win-64     621 KB
2021-02-21T02:12:07.8480370Z   libsodium                   1.0.18  h62dcd97_1           conda-forge/win-64     664 KB
2021-02-21T02:12:07.8481749Z   libssh2                      1.9.0  hb06d900_5           conda-forge/win-64     Cached
2021-02-21T02:12:07.8482462Z   libtiff                      4.2.0  hc10be44_0           conda-forge/win-64       1 MB
2021-02-21T02:12:07.8483477Z   libwebp-base                 1.2.0  h8ffe710_0           conda-forge/win-64     367 KB
2021-02-21T02:12:07.8484242Z   libxml2                     2.9.10  hf5bbc77_3           conda-forge/win-64     Cached
2021-02-21T02:12:07.8484914Z   libzip                       1.7.3  hfed4ece_0           conda-forge/win-64     134 KB
2021-02-21T02:12:07.8485592Z   lz4-c                        1.9.3  h8ffe710_0           conda-forge/win-64     Cached
2021-02-21T02:12:07.8486487Z   metis                        5.1.0  ha925a31_1006        conda-forge/win-64       4 MB
2021-02-21T02:12:07.8487223Z   mkl                         2020.4  hb70f87d_311         conda-forge/win-64     172 MB
2021-02-21T02:12:07.8488441Z   mumps-seq                    5.2.1  hb3f9cae_10          conda-forge/win-64       6 MB
2021-02-21T02:12:07.8489198Z   numpy                       1.20.1  py39h6635163_0       conda-forge/win-64       5 MB
2021-02-21T02:12:07.8490710Z   ogre                       1.10.12  hadbb816_7           conda-forge/win-64     113 MB
2021-02-21T02:12:07.8491527Z   opencv                       4.5.1  py39hcbf5309_0       conda-forge/win-64      22 KB
2021-02-21T02:12:07.8492373Z   openexr                      2.5.5  hab3b255_0           conda-forge/win-64       3 MB
2021-02-21T02:12:07.8492992Z   openssl                     1.1.1j  h8ffe710_0           conda-forge/win-64     Cached
2021-02-21T02:12:07.8494239Z   pip                         21.0.1  pyhd8ed1ab_0         conda-forge/noarch       1 MB
2021-02-21T02:12:07.8494860Z   portaudio                   19.6.0  hca4a3dc_3           conda-forge/win-64      89 KB
2021-02-21T02:12:07.8495535Z   protobuf                    3.14.0  py39h415ef7b_1       conda-forge/win-64     261 KB
2021-02-21T02:12:07.8496118Z   pugixml                     1.11.4  h0e60522_0           conda-forge/win-64     211 KB
2021-02-21T02:12:07.8496746Z   py-opencv                    4.5.1  py39h832f523_0       conda-forge/win-64      23 KB
2021-02-21T02:12:07.8497347Z   python                       3.9.1  h7840368_5_cpython   conda-forge/win-64      20 MB
2021-02-21T02:12:07.8498003Z   python_abi                     3.9  1_cp39               conda-forge/win-64       4 KB
2021-02-21T02:12:07.8498720Z   qt                          5.12.9  h5909a2a_4           conda-forge/win-64     106 MB
2021-02-21T02:12:07.8499476Z   qwt                          6.1.5  h265ddaf_0           conda-forge/win-64       4 MB
2021-02-21T02:12:07.8500583Z   ruby                         2.7.2  h8b1b97a_3           conda-forge/win-64      13 MB
2021-02-21T02:12:07.8501540Z   sdl                         1.2.15  h21ff451_1           conda-forge/win-64     248 KB
2021-02-21T02:12:07.8502121Z   sdl2                        2.0.12  h0e60522_1           conda-forge/win-64       2 MB
2021-02-21T02:12:07.8502682Z   setuptools                  52.0.0  py39haa95532_0       pkgs/main/win-64       725 KB
2021-02-21T02:12:07.8503342Z   six                         1.15.0  pyh9f0ad1d_0         conda-forge/noarch      14 KB
2021-02-21T02:12:07.8503914Z   sqlite                      3.34.0  h8ffe710_0           conda-forge/win-64       1 MB
2021-02-21T02:12:07.8504482Z   tbb                         2020.3  h74a9793_0           pkgs/main/win-64       156 KB
2021-02-21T02:12:07.8504986Z   tbb-devel                   2020.3  h74a9793_0           pkgs/main/win-64         2 MB
2021-02-21T02:12:07.8505683Z   tiny-process-library         2.0.4  h0e60522_0           conda-forge/win-64      24 KB
2021-02-21T02:12:07.8506337Z   tinyxml                      2.6.2  h2d74725_2           conda-forge/win-64      70 KB
2021-02-21T02:12:07.8506976Z   tinyxml2                     8.0.0  he025d50_1           conda-forge/win-64     120 KB
2021-02-21T02:12:07.8507533Z   tk                          8.6.10  he774522_1           conda-forge/win-64       3 MB
2021-02-21T02:12:07.8508373Z   tzdata                       2021a  he74cb21_0           conda-forge/noarch     121 KB
2021-02-21T02:12:07.8508975Z   urdfdom                      2.3.3  h7ef1ec2_0           conda-forge/win-64     121 KB
2021-02-21T02:12:07.8509599Z   urdfdom_headers              1.0.5  h7ef1ec2_2           conda-forge/win-64      16 KB
2021-02-21T02:12:07.8510886Z   wheel                       0.36.2  pyhd3deb0d_0         conda-forge/noarch      31 KB
2021-02-21T02:12:07.8511512Z   wincertstore                   0.2  py39hcbf5309_1006    conda-forge/win-64      15 KB
2021-02-21T02:12:07.8512158Z   xz                           5.2.5  h62dcd97_1           conda-forge/win-64     Cached
2021-02-21T02:12:07.8512695Z   yaml                         0.2.5  he774522_0           conda-forge/win-64      61 KB
2021-02-21T02:12:07.8513475Z   zeromq                       4.3.3  h0e60522_3           conda-forge/win-64       9 MB
2021-02-21T02:12:07.8514172Z   zstd                         1.4.8  h4e2f164_1           conda-forge/win-64     Cached
2021-02-21T02:12:07.8515021Z   zziplib                    0.13.69  ha81a0f5_1           conda-forge/win-64      50 KB
~~~

Packages installed by first not working: 
~~~
2021-02-22T02:12:17.2939216Z   ace                          7.0.0  h0e60522_1           conda-forge/win-64       2 MB
2021-02-22T02:12:17.2939808Z   asio                        1.14.1  h0e60522_1           conda-forge/win-64     327 KB
2021-02-22T02:12:17.2940728Z   boost                       1.74.0  py39hefe7e4c_3       conda-forge/win-64     570 KB
2021-02-22T02:12:17.2941566Z   boost-cpp                   1.74.0  h54f0996_2           conda-forge/win-64      16 MB
2021-02-22T02:12:17.2942187Z   bzip2                        1.0.8  h8ffe710_4           conda-forge/win-64     Cached
2021-02-22T02:12:17.2942924Z   ca-certificates          2021.1.19  haa95532_0           pkgs/main/win-64       122 KB
2021-02-22T02:12:17.2943732Z   certifi                  2020.12.5  py39hcbf5309_1       conda-forge/win-64     144 KB
2021-02-22T02:12:17.2944634Z   console_bridge               1.0.1  h5362a0b_0           conda-forge/win-64      23 KB
2021-02-22T02:12:17.2945258Z   cppzmq                       4.7.1  h50e1b43_1           conda-forge/win-64      26 KB
2021-02-22T02:12:17.2945835Z   curl                        7.71.1  h4b64cdc_8           conda-forge/win-64     122 KB
2021-02-22T02:12:17.2946461Z   dlfcn-win32                  1.2.0  he025d50_0           conda-forge/win-64      21 KB
2021-02-22T02:12:17.2947071Z   eigen                        3.3.9  h2d74725_1           conda-forge/win-64     970 KB
2021-02-22T02:12:17.2947955Z   ffmpeg                       4.3.1  ha925a31_0           conda-forge/win-64      26 MB
2021-02-22T02:12:17.2948522Z   freeglut                     3.2.1  h0e60522_2           conda-forge/win-64     149 KB
2021-02-22T02:12:17.2949127Z   freeimage                   3.18.0  h33f27b4_0           conda-forge/win-64       2 MB
2021-02-22T02:12:17.2949709Z   freetype                    2.10.4  h546665d_1           conda-forge/win-64     489 KB
2021-02-22T02:12:17.2950298Z   gazebo                      11.3.0  h4f5d396_6           conda-forge/win-64      62 MB
2021-02-22T02:12:17.2951012Z   glew                         2.1.0  h39d44d4_2           conda-forge/win-64     941 KB
2021-02-22T02:12:17.2951577Z   glfw                         3.3.2  h2fa13f4_1           conda-forge/win-64     118 KB
2021-02-22T02:12:17.2952121Z   gsl                            2.6  hdfb1a43_2           conda-forge/win-64       2 MB
2021-02-22T02:12:17.2953218Z   gts                          0.7.6  h39c6fae_2           conda-forge/win-64     188 KB
2021-02-22T02:12:17.2953884Z   hdf5                        1.10.6  nompi_h5268f04_1114  conda-forge/win-64      20 MB
2021-02-22T02:12:17.2954469Z   icu                           68.1  h0e60522_0           conda-forge/win-64      16 MB
2021-02-22T02:12:17.2955037Z   ilmbase                      2.5.5  h12d4b20_0           conda-forge/win-64     294 KB
2021-02-22T02:12:17.2955877Z   intel-openmp                2020.3  h57928b3_311         conda-forge/win-64       2 MB
2021-02-22T02:12:17.2956502Z   ipopt                       3.13.3  hf6be2e5_8           conda-forge/win-64       1 MB
2021-02-22T02:12:17.2957075Z   irrlicht                     1.8.4  hb2b2432_0           conda-forge/win-64       1 MB
2021-02-22T02:12:17.2957665Z   jasper                      2.0.14  h77af90b_2           conda-forge/win-64     662 KB
2021-02-22T02:12:17.2958225Z   jpeg                            9d  he774522_0           conda-forge/win-64     344 KB
2021-02-22T02:12:17.2961331Z   jsoncpp                      1.9.4  h2d74725_1           conda-forge/win-64     963 KB
2021-02-22T02:12:17.2962011Z   krb5                        1.17.2  hbae68bd_0           conda-forge/win-64     Cached
2021-02-22T02:12:17.2962774Z   libblas                      3.9.0  8_mkl                conda-forge/win-64       4 MB
2021-02-22T02:12:17.2963524Z   libcblas                     3.9.0  8_mkl                conda-forge/win-64       4 MB
2021-02-22T02:12:17.2964323Z   libclang                    11.0.1  default_h5c34c98_1   conda-forge/win-64      21 MB
2021-02-22T02:12:17.2964934Z   libcurl                     7.71.1  h4b64cdc_8           conda-forge/win-64     Cached
2021-02-22T02:12:17.2965611Z   libignition-cmake2           2.6.2  h0e60522_1           conda-forge/win-64     175 KB
2021-02-22T02:12:17.2966336Z   libignition-common3         3.10.1  he3878d5_0           conda-forge/win-64     545 KB
2021-02-22T02:12:17.2967322Z   libignition-fuel-tools4      4.3.0  h0ee6c32_1           conda-forge/win-64     211 KB
2021-02-22T02:12:17.2968086Z   libignition-math6            6.7.0  h0e60522_1           conda-forge/win-64     198 KB
2021-02-22T02:12:17.2968803Z   libignition-msgs5            5.3.0  h0e60522_5           conda-forge/win-64       2 MB
2021-02-22T02:12:17.2969618Z   libignition-tools1           1.0.0  h57928b3_3           conda-forge/win-64       8 KB
2021-02-22T02:12:17.2970575Z   libignition-transport8       8.1.0  h7435cc3_3           conda-forge/win-64     326 KB
2021-02-22T02:12:17.2971319Z   libjpeg-turbo                2.0.5  he774522_0           conda-forge/win-64       1 MB
2021-02-22T02:12:17.2971922Z   liblapack                    3.9.0  8_mkl                conda-forge/win-64       4 MB
2021-02-22T02:12:17.2972519Z   liblapacke                   3.9.0  8_mkl                conda-forge/win-64       4 MB
2021-02-22T02:12:17.2973282Z   libmatio                    1.5.19  h216f7ef_0           conda-forge/win-64     156 KB
2021-02-22T02:12:17.2973872Z   libode                        0.16  h6538335_0           conda-forge/win-64     311 KB
2021-02-22T02:12:17.2974453Z   libopencv                    4.5.1  py39h27d8466_0       conda-forge/win-64      46 MB
2021-02-22T02:12:17.2975062Z   libpng                      1.6.37  ha81a0f5_2           conda-forge/win-64     763 KB
2021-02-22T02:12:17.2975788Z   libprotobuf                 3.14.0  h7755175_0           conda-forge/win-64       2 MB
2021-02-22T02:12:17.2976435Z   libsdformat                  9.3.0  h4bf3a07_2           conda-forge/win-64     621 KB
2021-02-22T02:12:17.2977032Z   libsodium                   1.0.18  h62dcd97_1           conda-forge/win-64     664 KB
2021-02-22T02:12:17.2977645Z   libssh2                      1.9.0  hb06d900_5           conda-forge/win-64     Cached
2021-02-22T02:12:17.2978229Z   libtiff                      4.2.0  hc10be44_0           conda-forge/win-64       1 MB
2021-02-22T02:12:17.2979020Z   libwebp-base                 1.2.0  h8ffe710_0           conda-forge/win-64     367 KB
2021-02-22T02:12:17.2979646Z   libxml2                     2.9.10  hf5bbc77_3           conda-forge/win-64     Cached
2021-02-22T02:12:17.2980236Z   libzip                       1.7.3  hfed4ece_0           conda-forge/win-64     134 KB
2021-02-22T02:12:17.2980851Z   lz4-c                        1.9.3  h8ffe710_0           conda-forge/win-64     Cached
2021-02-22T02:12:17.2981419Z   metis                        5.1.0  ha925a31_1006        conda-forge/win-64       4 MB
2021-02-22T02:12:17.2982118Z   mkl                         2020.4  hb70f87d_311         conda-forge/win-64     172 MB
2021-02-22T02:12:17.2982916Z   mumps-seq                    5.2.1  hb3f9cae_10          conda-forge/win-64       6 MB
2021-02-22T02:12:17.2983662Z   numpy                       1.20.1  py39h6635163_0       conda-forge/win-64       5 MB
2021-02-22T02:12:17.2984225Z   ogre                       1.10.12  hadbb816_7           conda-forge/win-64     113 MB
2021-02-22T02:12:17.2985015Z   opencv                       4.5.1  py39hcbf5309_0       conda-forge/win-64      22 KB
2021-02-22T02:12:17.2985741Z   openexr                      2.5.5  hab3b255_0           conda-forge/win-64       3 MB
2021-02-22T02:12:17.2986329Z   openssl                     1.1.1j  h8ffe710_0           conda-forge/win-64     Cached
2021-02-22T02:12:17.2986905Z   pip                         21.0.1  pyhd8ed1ab_0         conda-forge/noarch       1 MB
2021-02-22T02:12:17.2987486Z   portaudio                   19.6.0  hca4a3dc_3           conda-forge/win-64      89 KB
2021-02-22T02:12:17.2988213Z   protobuf                    3.14.0  py39h415ef7b_1       conda-forge/win-64     261 KB
2021-02-22T02:12:17.2988787Z   pugixml                     1.11.4  h0e60522_0           conda-forge/win-64     211 KB
2021-02-22T02:12:17.2989340Z   py-opencv                    4.5.1  py39h832f523_0       conda-forge/win-64      23 KB
2021-02-22T02:12:17.2989931Z   python                       3.9.2  h7840368_0_cpython   conda-forge/win-64      20 MB
2021-02-22T02:12:17.2990622Z   python_abi                     3.9  1_cp39               conda-forge/win-64       4 KB
2021-02-22T02:12:17.2991124Z   qt                          5.12.9  h5909a2a_4           conda-forge/win-64     106 MB
2021-02-22T02:12:17.2991640Z   qwt                          6.1.5  h265ddaf_0           conda-forge/win-64       4 MB
2021-02-22T02:12:17.2992220Z   ruby                         2.7.2  h8b1b97a_3           conda-forge/win-64      13 MB
2021-02-22T02:12:17.2996351Z   sdl                         1.2.15  h21ff451_1           conda-forge/win-64     248 KB
2021-02-22T02:12:17.2997040Z   sdl2                        2.0.12  h0e60522_1           conda-forge/win-64       2 MB
2021-02-22T02:12:17.2997636Z   setuptools                  52.0.0  py39haa95532_0       pkgs/main/win-64       725 KB
2021-02-22T02:12:17.2998225Z   six                         1.15.0  pyh9f0ad1d_0         conda-forge/noarch      14 KB
2021-02-22T02:12:17.2998832Z   sqlite                      3.34.0  h8ffe710_0           conda-forge/win-64       1 MB
2021-02-22T02:12:17.2999356Z   tbb                         2020.3  h74a9793_0           pkgs/main/win-64       156 KB
2021-02-22T02:12:17.2999886Z   tbb-devel                   2020.3  h74a9793_0           pkgs/main/win-64         2 MB
2021-02-22T02:12:17.3000535Z   tiny-process-library         2.0.4  h0e60522_0           conda-forge/win-64      24 KB
2021-02-22T02:12:17.3001208Z   tinyxml                      2.6.2  h2d74725_2           conda-forge/win-64      70 KB
2021-02-22T02:12:17.3001770Z   tinyxml2                     8.0.0  he025d50_1           conda-forge/win-64     120 KB
2021-02-22T02:12:17.3002345Z   tk                          8.6.10  he774522_1           conda-forge/win-64       3 MB
2021-02-22T02:12:17.3002889Z   tzdata                       2021a  he74cb21_0           conda-forge/noarch     121 KB
2021-02-22T02:12:17.3003671Z   urdfdom                      2.3.3  h7ef1ec2_0           conda-forge/win-64     121 KB
2021-02-22T02:12:17.3004302Z   urdfdom_headers              1.0.5  h7ef1ec2_2           conda-forge/win-64      16 KB
2021-02-22T02:12:17.3004922Z   wheel                       0.36.2  pyhd3deb0d_0         conda-forge/noarch      31 KB
2021-02-22T02:12:17.3005583Z   wincertstore                   0.2  py39hcbf5309_1006    conda-forge/win-64      15 KB
2021-02-22T02:12:17.3006268Z   xz                           5.2.5  h62dcd97_1           conda-forge/win-64     Cached
2021-02-22T02:12:17.3006933Z   yaml                         0.2.5  he774522_0           conda-forge/win-64      61 KB
2021-02-22T02:12:17.3007440Z   zeromq                       4.3.3  h0e60522_3           conda-forge/win-64       9 MB
2021-02-22T02:12:17.3007986Z   zstd                         1.4.8  h4e2f164_1           conda-forge/win-64     Cached
2021-02-22T02:12:17.3008523Z   zziplib                    0.13.69  ha81a0f5_1           conda-forge/win-64      50 KB
~~~

**Comment by traversaro** [comment]
The main difference seems `ignition-msgs5 5.3.0  h0e60522_4` was installed when it was working, while `ignition-msgs5 5.3.0  h0e60522_5` was not working.

**Comment by traversaro** [comment]
The main problem is that `ignition-msgs5 5.3.0 h0e60522_5` is compiled with libprotobuf 3.15 but it has no constraint on the protobuf version installed due to the issue fixed in https://github.com/conda-forge/libignition-msgs1-feedstock/pull/28 . The proper way to fix this is patch that at the repodata level, but as discussed in https://github.com/conda-forge/libignition-msgs1-feedstock/pull/28#issuecomment-783709868 we are not sure how to do so. This problem is tracked in https://github.com/conda-forge/libignition-msgs1-feedstock/issues/30 .

However, I was not super-worried about https://github.com/conda-forge/libignition-msgs1-feedstock/issues/30 as if the protobuf315 migration is completed for our packages, protobuf315 should be installed instead of 3.15. However, this seems not to be happening, not sure why.

**Comment by traversaro** [comment]
The route problem seem that an old Gazebo 11.0.0 is installed for some reason. I reproduced the issue locally, where:
~~~
mamba install gazebo
~~~
installs the latest gazebo (11.3.0 build 7), while 
~~~
mamba install ace asio boost eigen gazebo
~~~
installs gazebo 11.0.0 .

**Comment by traversaro** [comment]
The problem is boost, as if i do:
~~~
mamba install boost gazebo
~~~

boost 1.75 and gazebo 11.0.0 build 3 are installed, while if I do:
~~~
mamba install gazebo boost
~~~

boost 1.74 and gazebo 11.3.0 build 7 are installed.

Interestingly, if I use conda and I do:
~~~
conda install gazebo boost
~~~
or 
~~~
conda install boost gazebo
~~~

In both cases I get boost 1.74 and gazebo 11.3.0 build 7 . So this may be a small difference in conda and mamba.

**Comment by traversaro** [comment]
I investigate the root cause of the problem that I am seeing now (that may not be the problem that has broken the CI since 5 days), and it seems that boost 1.75 was just released, and by inspecting in the `index.json` file of `gazebo 11.0.0 build 3` it just contain the line:
~~~
    "boost-cpp >=1.71",
~~~
instead of the correct one:
~~~
    "boost-cpp >=1.71.0,<1.71.1.0a0",
~~~
So gazebo 11.0.0 build 3 thinks that is compatible with boost 1.75, but it does not. The problem only affects Windows as  the 11.0.0 build 3 of Gazebo was only available for Windows, and the first version available for Linux is 11.0.0 build 5 that has the correct dependency for Boost:
~~~
    "boost-cpp >=1.72.0,<1.72.1.0a0",
~~~

**Comment by traversaro** [comment]
> 
> 
> I investigate the root cause of the problem that I am seeing now (that may not be the problem that has broken the CI since 5 days), and it seems that boost 1.75 was just released, and by inspecting in the `index.json` file of `gazebo 11.0.0 build 3` it just contain the line:
> 
> ```
>     "boost-cpp >=1.71",
> ```
> 
> instead of the correct one:
> 
> ```
>     "boost-cpp >=1.71.0,<1.71.1.0a0",
> ```
> 
> So gazebo 11.0.0 build 3 thinks that is compatible with boost 1.75, but it does not. The problem only affects Windows as the 11.0.0 build 3 of Gazebo was only available for Windows, and the first version available for Linux is 11.0.0 build 5 that has the correct dependency for Boost:
> 
> ```
>     "boost-cpp >=1.72.0,<1.72.1.0a0",
> ```

This problem should have been fixed by https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/126 . Let's wait for the CDN to propagate the change, and let's see if the problem is still there.

**Comment by traversaro** [comment]
As you can see the CI in https://github.com/robotology/robotology-superbuild/pull/640/checks?check_run_id=1995513492 , the problem was indeed solved by https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/126 .
